src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -65,106 +63,6 @@
 from llmcompressor.utils.fsdp.helpers import is_fsdp_model
 
 
-def train(**kwargs):
```

Comment:
This is also used for the entrypoints for python api, so needed

Reply:
Not all, but I will clean this up and confirm.